### PR TITLE
add EscapeAPI to security

### PR DESCRIPTION
**Pull request by polarspetroll**
- [X] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [X] My addition is ordered alphabetically
- [X] My submission has a useful description
- [X] The description does not end with punctuation
- [X] Each table column is padded with one space on either side
- [X] I have searched the repository for any relevant issues or pull requests
- [X] Any category I am creating has the minimum requirement of 3 items
- [X] All changes have been [squashed][squash-link] into a single commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -864,6 +864,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Censys.io](https://censys.io/api) | Search engine for Internet connected host and devices | `apiKey` | Yes | No |
 | [CRXcavator](https://crxcavator.io/apidocs) | Chrome extension risk scoring | `apiKey` | Yes | Unknown |
+| [Escape](https://github.com/polarspetroll/EscapeAPI)| an API for escaping different kind of queries | No | Yes | No
```
Comment (matheusfelipeog) [inline README.md:867]: - Put a blank space between the link and `|` character
- Start the description with a capital letter
- And at the end of the line put a `|`

```suggestion
| [Escape](https://github.com/polarspetroll/EscapeAPI) | An API for escaping different kind of queries | No | Yes | No |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @polarspetroll ✋ 

Can you resolve the review, please?

**Comment by polarspetroll** [comment]
Hi, Thanks for your reply . Changes committed successfully

**Review by matheusfelipeog — APPROVED** [review]
Perfect ❤️ 

A big THANKS, we are grateful for your update.

**Pull request merged**